### DISCUSSION TITLE
Issues creating a Dockerfile

### DISCUSSION THREAD

**Discussion by norwin2012** [Pythagora-io/gpt-pilot#547] [Q&A]
I've noticed an issue with the program in the vscode plug-in when creating a dockerfile it creates it as a directory instead of a .txt equivalent. I'm not exactly code savvy to find the script called for file creation/generation. Would anyone know where I can find this file so I might be able to change the logic some to create it as dockerfile.txt then remove the extension as needed?

**Comment by alva333** [comment]
did you figure it out

**Reply by norwin2012** [reply]
I have not.
